src/network/connectionserviceprovider.cpp

```diff
@@ -0,0 +1,13 @@
+#include "connectionserviceprovider.h"
+
+ConnectionServiceProvider::ConnectionServiceProvider(QObject *parent) : QObject(parent)
+{
+    m_apiParser = DefaultLocalConnector::generateConnector();
+
```

Comment:
remove empty line

Reply:
This is outdated file too. It is removed in the latest commit.